### DISCUSSION TITLE
Adding Spanish translation

### DISCUSSION THREAD

**Discussion by kcholoren** [0xJacky/nginx-ui#117] [Q&A] [answered]
Hello, I saw PR115 about the French translation and following your instructions I completed the Spanish translation.
I forked your project and want to open a pull request, but the French translation hasn't been merged yet.
Do you prefer to wait or pull now?

Thank you.

**Comment by 0xJacky** [comment]
Hi, French translation has been merged into `dev` branch, you can fork this branch and add translation into it.

**Comment by Hintay** [comment, marked as answer]
Hello! Thank you for your contribution and for completing the Spanish translation, we appreciate your efforts in helping us make the project accessible to more people.

Regarding the French translation, we have already merged it into the `dev` branch. You can synchronize your forked repository with the `dev` branch and then proceed to create a pull request. This way, any conflicts can be resolved during the merging process.

Alternatively, if you prefer, you can create the pull request now, and we will take care of resolving any conflicts that may arise.

Thank you again for your contributions, and please let us know if you have any further questions or need any assistance.

**Comment by kcholoren** [comment]
Hello, thank you very much for the answer. Maybe in the future I can review the English language because translating into Spanish was complicated in some sentences.  I ended up doing it by looking at where they were used in the source code, the frontend, and the Chinese language version.
